src/tuning.ts

```diff
@@ -19,59 +24,63 @@ export class Tuning {
 
   constructor(tuningString = 'E/5,B/4,G/4,D/4,A/3,E/3,B/2,E/2') {
     // Default to standard tuning.
+    this.tuningString = '';
+    this.tuningValues = [];
+    this.numStrings = 0;
     this.setTuning(tuningString);
   }
 
-  noteToInteger(noteString) {
+  static noteToInteger(noteString: string): number {
```

Comment:
@0xfe @mscuthbert proposals on comments of functions in this file is appreciated.